scripts/validate.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# do basic validations about AMI generated
```

Comment:
Nit: `Do basic validation of the generated AMI`

Reply:
fixed 👀